RELEASE/scripts/autoscend/quests/level_11.ash

```diff
@@ -984,6 +984,15 @@ boolean L11_getUVCompass()
 	return false;
 }
 
+boolean L11_hasUltrahydrated()
```

Comment:
Should this also return true when we have the survival knife (and it is usable)?